### ISSUE TITLE
intersect argument order

### ISSUE THREAD

**Issue by ioRekz** [Turfjs/turf#1731]
```javascript
var x = polygon([[[1, 0], [1, 3], [3, 3], [3, 0], [1, 0]]])
var y = polygon([[[2, 0], [2, 3], [3, 3], [3, 0], [2, 0]]])
console.log(intersect(x, y)) //null
console.log(intersect(y, x)) //[Feature]
```
Using version 6.0.1 of turf/intersect
The first console.log will print null. The other will print a Feature object.
As I understand it, the order of params should not matter for calculating the intersection.
 
Using the older 5.1.6 version:
It will print 2 similar Feature object as I expected (with different coordinates order).

- [x] Snippet of source code or for complex examples use [jsfiddle](https://jsfiddle.net/).
Here is a codesandbox example 
https://codesandbox.io/embed/affectionate-burnell-uv38l
You can try switching from version to version to see when it differs.

**Comment by dpmcmlxxvi** [comment]
@ioRekz  For the polygon case, Turf is just trivially passing the features to the `martinez-polygon-clipping` library. It looks like they are already tracking this issue [here](https://github.com/w8r/martinez/issues/95). 

However, the main developer recently tagged all the issues with `help wanted`, so it may not get fixed anytime soon if that repo is no longer a top priority for them.

**Comment by rowanwins** [comment]
Resolved with v6.3.0
